src/views/main-view.js

```diff
@@ -80,11 +80,14 @@ MainView.prototype._initialize = function () {
 
   this.model.on('changeActivePaymentView', function (id) {
     if (id === PaymentMethodsView.ID) {
-      classlist.add(paymentMethodsViews.element, 'braintree-methods--active');
+      classlist.add(paymentMethodsViews.container, 'braintree-methods--active');
+      classlist.remove(paymentMethodsViews.container, 'braintree-methods-container--inactive');
```

Comment:
Does `braintree-methods-container--inactive` get added back in the case that the transition needs to happen again? Didn't see it but I could just be missing it.